recipes/zlib/bld.bat

```diff
@@ -0,0 +1,24 @@
+:: Configure step.
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      - DCMAKE_INSTALL_PREFIX:%LIBRARY_PREFIX% %SRC_DIR% ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build step.
+cmake --build %SRC_DIR% --target INSTALL --config Release
+if errorlevel 1 exit 1
+
+:: cmake Test.
+ctest
+if errorlevel 1 exit 1
```

Comment:
Yeah, but even Linus shell scripts need an option for this.  I think it is 

    set -e

?  I don't know if windows has an equivalent.  Basically this practice prevents you from ever uploading half-baked packages.